### DISCUSSION TITLE
How to use Automapper for call methods for set properties?

### DISCUSSION THREAD

**Discussion by P9avel** [LuckyPennySoftware/AutoMapper#4008] [NA]
Hello. I am have code
 
var payment = new Payment(Guid.NewGuid())
                .SetUserId(request.UserId)
                .SetDateTime(request.DateTime)
                .SetAmount(request.Amount)
                .SetDescription(request.Description)
                .SetReceiveDateTime(DateTime.UtcNow);

i am want to tuning automapper for calling  var payment = _mapper.Map<Payment>(request);
Can you to show Profile example for my case?

**Comment by lbargaoanu** [comment]
AM doesn't work that way. Destination values are set using properties, fields or constructors (public or not).
